packages/input-mask/README.md

```diff
@@ -0,0 +1,82 @@
+# @solid-primitives/input-mask
+
+[![lerna](https://img.shields.io/badge/maintained%20with-lerna-cc00ff.svg?style=for-the-badge)](https://lerna.js.org/)
+[![size](https://img.shields.io/bundlephobia/minzip/@solid-primitives/input-mask?style=for-the-badge&label=size)](https://bundlephobia.com/package/@solid-primitives/input-mask)
+[![version](https://img.shields.io/npm/v/@solid-primitives/input-mask?style=for-the-badge)](https://www.npmjs.com/package/@solid-primitives/input-mask)
+[![stage](https://img.shields.io/endpoint?style=for-the-badge&url=https%3A%2F%2Fraw.githubusercontent.com%2Fdavedbase%2Fsolid-primitives%2Fmain%2Fassets%2Fbadges%2Fstage-0.json)](https://github.com/davedbase/solid-primitives#contribution-process)
+
+Primitive that returns an event handler to mask the inputs of an text input element (`<input>`, `<textarea>`) when applied in `oninput` or `onchange`.
+
+## Installation
+
+```bash
+npm install @solid-primitives/input-mask
+# or
+yarn add @solid-primitives/input-mask
+```
+## Usage
+
+The masks come in 3 different formats: function, array and string. There are tools to convert string masks to array masks and array masks to function masks.
+
+```ts
+// string mask:
```

Comment:
Including imports is always nice to know what's coming from the package, and how to import it.
```suggestion
import { createInputMask } from "@solid-primitives/input-mask";
import { anyMaskToFn} from "@solid-primitives/input-mask";

// string mask:
```

Reply:
Then I should probably include all of them.